### ISSUE TITLE
String based enums access via Namespace Import leads to runtime errors

### ISSUE THREAD

**Issue by yoosiba** [NumberFour/n4js#386]
It seems that transpiler is not replacing access to string based enums via namespace imports correctly

```typescript
// module Defs
export public enum X{
	X1 : "x one"
}

@StringBased
export public enum Y{
	Y1 : "y one"
}

//caller module
import * as N from "Defs";

console.log(N.X.X1) // ok
console.log(N.Y.Y1) // error at runtime
```
runtime output
```
X { name: 'X1', value: 'x one' }
(SystemJS) Cannot read property 'Y1' of undefined
	TypeError: Cannot read property 'Y1' of undefined
	    at execute (/Users/me/work/workspace/MyProj/src-gen/es/MyProj/Caller.js:15:24)
	Error loading MyProj/Caller
```

looks like transpiler is not inlining enum when accessed via namespace
```typescript
			execute: function() {
				console.log(N.X.X1);
				console.log(N.Y.Y1); // value should be inlined, i.e. console.log("y one");
			}
```

**Comment by yoosiba** [comment]
@arrowplum yup, it is a bug

**Comment by yoosiba** [comment]
fix to this will be affected by fix for #387

**Comment by yoosiba** [comment]
fixed with https://github.com/NumberFour/n4js/commit/9c33b4d9335f9f33137c51a91eb7538a14698e8e
